### PR TITLE
Provide access to DRONE_DEPLOY_TO

### PR DESCRIPTION
**Pull request by daniel-ac-martin**
Provides access to the `DRONE_DEPLOY_TO` variable via
`{{build.deployTo}}` in a custom template. This allows one to include
which environment has been depoyed to when using deployments in Drone.

Addresses #51.

### CODE REVIEW

**Comment by daniel-ac-martin** [comment]
I get a blank page from Drone so I can't see what, if anything has gone wrong.

I've tested this locally using the following command:
```shell
$ docker run --rm \
    -e SLACK_WEBHOOK=https://hooks.slack.com/services/... \
    -e PLUGIN_CHANNEL=general \
    -e PLUGIN_USERNAME=drone-dacm \
    -e DRONE_REPO_OWNER=octocat \
    -e DRONE_REPO_NAME=hello-world \
    -e DRONE_COMMIT_SHA=7fd1a60b01f91b314f59955a4e4d4e80d8edf11d \
    -e DRONE_COMMIT_BRANCH=master \
    -e DRONE_COMMIT_AUTHOR=octocat \
    -e DRONE_BUILD_NUMBER=1 \
    -e DRONE_BUILD_STATUS=success \
    -e DRONE_BUILD_LINK=http://github.com/octocat/hello-world \
    -e DRONE_DEPLOY_TO=preprod \
    -e DRONE_TAG=1.0.0 \
    -e PLUGIN_TEMPLATE='|{{build.deployTo}}|' \
    plugins/slack
```

File: main.go [hunk 1]
```diff
@@ -132,6 +132,11 @@ func main() {
 			Usage:  "build tag",
 			EnvVar: "DRONE_TAG",
 		},
+		cli.StringFlag{
+			Name:   "build.deployTo",
```
Comment (tboerger) [inline main.go]: @bradrydzewski is this really a name we already got elsewhere?

**Comment by daniel-ac-martin** [comment]
@bradrydzewski: Could you let me know whether there is anything wrong with this PR? I would really like to get this to the point where it can be merged in.

**Comment by daniel-ac-martin** [comment]
Noticed this branch was out of date, so I've done a `git pull --rebase` on it.

**Pull request merged**